core/src/utils/animation/animation-utils.ts

```diff
@@ -1,9 +1,25 @@
+export const getAnimationPrefix = (el: HTMLElement): string => {
+  const supportsUnprefixed = (el.style as any).animationName !== undefined;
+  const supportsWebkitPrefix = (el.style as any).webkitAnimationName !== undefined;
+
+  if (!supportsUnprefixed && supportsWebkitPrefix) {
+    return '-webkit-';
+  }
+
+  return '';
+};
 
 export const setStyleProperty = (element: HTMLElement, propertyName: string, value: string | null) => {
+  if (propertyName.startsWith('animation')) {
```

Comment:
I'll leave this up to you, but I typically try to avoid modifying the values of function parameters (I treat them as if they are `const` except for very specific use cases).

So instead you could just do:

```
element.style.setProperty(
  propertyName.startsWith('animation') ? animationPrefix + propertyName : propertyName, 
  value
);
```

Again, totally up to you though.

Reply:
I can change this.